### ISSUE TITLE
go through tests and remove engine crashes?

### ISSUE THREAD

**Issue by increpare** [increpare/PuzzleScript#906]
probably a good idea...

**Comment by david-pfx** [comment]
Do you have a list?

**Comment by increpare** [comment]
No, I amn't even sure if there are any engine crashes.  The plan was to just run the tests (as described in https://github.com/increpare/PuzzleScript/blob/master/DEVELOPMENT.md ) and double-check there are no engine crashes somehow.

**Comment by increpare** [comment]
seems good now
